### PR TITLE
Mobile menu popup

### PR DESCRIPTION
**Pull request by Afsheen96**
In mobile, implement the following features:

 add the [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js).
 When the user clicks (or taps) the menu button, the mobile menu appears.
 When the user clicks (or taps) the close (X) button, the mobile menu disappears.
 When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
 When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -58,7 +58,42 @@ ul {
   margin-right: 10px;
 }
 
-nav ul {
+.mobile-menu {
+  background: #6070ff;
+  position: absolute;
```
Comment (zieeco) [inline style.css]: - Hey, kindly make this `fixed` so that the height can take up the whole height of the device to improve your design.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Afsheen96,





Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Mobile menu works correctly.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions


- N/A
 
**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Afsheen96) [reply style.css]: kindly tell me do you want me to fix the position @zieeco

Comment (zieeco) [reply style.css]: @Afsheen96, change the position from absolute to fixed.

Comment (Afsheen96) [reply style.css]: i did it thanks

**Review by omarsalem7 — APPROVED** [review]
Hi @Afsheen96 😊, 
# STATUS: APPROVED! 🥇 💯 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**